setup.cfg

```diff
@@ -17,7 +17,7 @@ select = E,F,W
 ignore = E123,E126,E203,E226,E241,E704,W503,W504
 
 [pydocstyle]
-match = ^((?!\.sync-zenodo-metadata|setup|benchmark).)*\.py$
+match = ^((?!\.sync-zenodo-metadata|setup).)*\.py$
```

Comment:
```suggestion
match = ^((?!\.sync-zenodo-metadata|setup|benchmark).)*\.py$
```

Reply:
We need to leave this ignore in so that pydocstyle doesn't look at the asv benchmarks.